README.md

```diff
@@ -6,6 +6,17 @@ troubleshooting the host operating system, without having to install software
 on the host. It is built on top of [Podman](https://podman.io/) and other
 standard container technologies from [OCI](https://opencontainers.org/).
 
+> [!NOTE]
+> **Toolbx's original name was "toolbox".** Work is still in progress to
+> update the old name to the new in various places, thus the name of the
+> package and the binary may still be `toolbox` on various systems.
+>
+> If you'd like to help, see the commit log for [45a9b40][] for a to-do list.
+>
+> The namespace change was undertaken for better discoverability, and "Toolbx"
+> with a capital "T" is now considered canonical; see [issue #1399][iss1399]
+> for a discussion.
```

Comment:
I would drop the second part about the discussion, because:

* It might encourage people to continue discussing this topic, and doing that will take away time that can be devoted to finishing the rename or reviewing the pending PRs or addressing the open issues, etc..

* The longer the note about the name, the more it pushes down the what Toolbx containers have access to and the security expectations around them, and some folks are really sensitive about that.  :/

Reply:
Seems fine. This was just a shot in the dark, and I was counting on feedback. I myself, being somewhat constitutionally anxious about spelling and pronouncing things correctly, actually _like_ those FAQ entries on project homepages which clarify spellings, pronunciations, and the origins of names. Hence, this blurb.